packages/google_atlas/test/widget_tests/google_atlas_widget_test.dart

```diff
@@ -192,6 +276,66 @@ main() {
       expect(platformGoogleMap.markerIdsToRemove.isEmpty, true);
     });
 
+    testWidgets(
+        'should return correct GoogleMap when build is called with multiple circles',
+        (WidgetTester tester) async {
+      final GoogleMaps.Circle expectedFirstCircle = GoogleMaps.Circle(
```

Comment:
All the types after `final` aren't needed I believe, since the compiler will infer them after you assign them.

Reply:
@jorgecoca we should apply `effective_dart` to `atlas`